consumers/hidden-test-consumer/src/hidden_test_consumer/consumer.py

```
@@ -0,0 +1,30 @@
+import os
+
+from base_consumer import BaseKafkaConsumer
+
+from hidden_test_consumer.process import HiddenTestProcessor
+
+
+class HiddenTestConsumer(BaseKafkaConsumer):
+    """Kafka consumer for handling hidden test messages."""
+
+    def __init__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)